src/js/rebuses.js

```diff
@@ -887,7 +887,12 @@ const rebuses = [
     symbols: ['🍯', '+', '🐝'],
     words: ['honeybee'],
     hint: ['An insect that makes a sweet treat.']
-  }
+  },
+  {
+    symbols: ['😇', '+', 'MAS'],
+    words: ['christmas'],
+    hint: ['A holiday we all like']
+  },
```

Comment:
I think that the travis integration tests failed because of the trailing comma, as it is not needed at the end of Array